crypto/pkcs11/pkcs11helpers.go

```diff
@@ -333,7 +331,7 @@ func privateDecryptOAEP(privKeyObj *Pkcs11KeyFileObject, ciphertext []byte, hash
 
 	var oaep *pkcs11.OAEPParams
 
-	// the default is sha1
+	// An empty string from the Hash in the JSON defaults to sha1.
```

Comment:
nit: looks like the file originally was not `go fmt`'ed, indent is weird, but can fix in another PR. just noting it.

Reply:
I don't see `gofmt -w -s` reformatting anything on this file.

Reply:
weird, maybe github formatting is doing weird things for me,